base/changes.txt

```diff
@@ -6,6 +6,11 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2023-12-06  Yukai Chou <muzimuzhi@gmail.com>
+	* ltcmd.dtx (subsection{Normalizing the argument specifications})
+	Use a clarified error message when an argument prefix `!' is applied to
```

Comment:
I've approved in review, but 'Clarify error message when ...' would be a little more natural

Reply:
Oh I forgot to update all these places. Thanks.